Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/FileTransporter.cs

```diff
@@ -318,16 +333,16 @@ private string GetUserResultMessage(IList<SingleFileTransferResult> results)
 
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/8c4346a7a0aded599281b906d6700bd7508cc79c/Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/FileTransporter.cs#L330
This method can be `static`.